packages/server/src/server.ts

```diff
@@ -197,26 +197,29 @@ app.getKoaApp().use(async (ctx, next) => {
   return next();
 });
 
-const logAndHandleResponse = async (methodName: any, methodParams: any, methodFunction: any) => {
-  const requestId = app.getRequestId();
-  const requestIdPrefix = requestId ? formatRequestIdMessage(requestId) : '';
+const logAndHandleResponse = async (methodName: string, methodParams: any[], methodFunction: any) => {
+  const requestDetails = app.getRequestDetails();
+  logger.debug('Method name: ' + methodName);
```

Comment:
Squash this into 1 line if needed.
That being said do we need it?
We should have a log that covers the method name no as well as the result

Reply:
yeah this was needed for debugging purposes only. will remove it